src/main/java/seedu/cakecollate/logic/LogicManager.java

```diff
@@ -54,6 +57,28 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         return commandResult;
     }
 
+    /**
+     * Updates the deliveryStatus to delivered if the delivery date is before the current date.
+     * @return A parsable string to update the deliveryStatus if necessary, and an empty string otherwise.
+     */
+    public String updateDeliveryStatus() {
```

Comment:
Very minor suggestion but you can consider renaming the method to updateDeliveryStatusToDelivered just to make the name more descriptive.